R/model_Collocations.R

```diff
@@ -86,7 +86,8 @@ Collocations = R6::R6Class(
     collocation_count_min = NULL,
     pmi_min = NULL,
     gensim_min = NULL,
-    lfmd_min = NULL
+    lfmd_min = NULL,
+    conf.level = NULL
```

Comment:
in the rest of the codebase we use `_` as separator, could you please adjust `.` here to match it?